src/main.ts

```diff
@@ -35,24 +36,30 @@ const stoa: Stoa = new Stoa(config.database.filename,
     config.server.address
     );
 
-stoa.createStorage().then(() => {
-    stoa.start().then(() => {
-        logger.info(`Listening to requests on: ${config.server.address}:${config.server.port}`);
+SodiumHelper.init().then(() => {
+    stoa.createStorage().then(() => {
```

Comment:
Does `createStorage` depends on Sodium working ?

Reply:
Sodium is WebAssembly.
It takes some waiting time for it to load into the mesh.
Although the createStorage is not dependent on sodium, the hash function is then used when blocks are received from the catch-up function.
I think it would be better to put the sodium work in the front rather than add it in the middle.